src/Microsoft.Azure.ServiceBus/Management/ManagementClient.cs

```diff
@@ -634,7 +634,7 @@ public virtual async Task<SubscriptionDescription> CreateSubscriptionAsync(Subsc
         /// <summary>
         /// Adds a new rule to the subscription under given topic.
         /// </summary>
-        /// <param name="topicName">The topic name relative to the service namespace base address.</param>
+        /// <param name="topicPath">The topic name relative to the service namespace base address.</param>
```

Comment:
Param name was changed, but not the description.

Reply:
Good catch, fixed.